Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,11 @@
 function sum(elements) {
-}
+        
+        let numbersOnly  = elements.filter(value => typeof value === 'number')
+        let sumOfNumbers = numbersOnly.reduce((acc,curr)=>acc+curr,0);
+         return sumOfNumbers;
+    
+    }
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function return the value you expected?

```
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```